metabase/metabase
# fix iframe dashcards crash subscriptions

### Description

Fixes iframe dashcards crash dashboard subscriptions by filtering out these cards.

### How to verify

- Create a dashboard with an iframe dashcard
- Send a test dashboard subscription 
- Ensure it does not fail

### Checklist
